.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ variables:
   build-release: &build-release
     run:
       name: Build and push release artifacts (e.g. docker containers)
-      command: RELEASE_CONFIRM=yes make push
+      command: RELEASE_CONFIRM=yes PYPI_PLATFORM=manylinux1_x86_64 make push
```

Comment:
While we don't support manylinux, as far as I can pypi will only let you upload packages tagged with manylinux. Tensorflow is our main dependency and they also tag their releases with manylinux but don't actually support manylinux. This is why we're held back from actually supporting manylinux. If a user can find a tensorflow distribution 1.9 or greater that works on their machine they should be able to use tfe.

Reply:
How'd we do this before now? I don't think we passed manylinux at all previously!

Reply:
We weren't explicitly platform specific before. But we were implicitly restricted to whatever tensorflow supports.

Reply:
O I think I understand your question now! I should be more specific. PYPI doesn't allow linux builds e.g. linux_x86_64 or something like ubuntu_x86_64. It only allows manylinux1_x86_64 for linux builds. We were just passing any platform before which was fine.

Reply:
Ah, got you! Is this documented someplace? Perhaps in the Makefile?